index.html

```diff
@@ -142,7 +142,7 @@ <h1 class="site-title">Company name</h1>
 							<small>18<sup>o</sup></small>
 						</div>
 					</div>
-					<div class="forecast">
+					<div id="f6" class="forecast">
```

Comment:
Liczę na to, że Git dobrze komentarz doda ;p patrząc na przykład linii 145 (i innych z ID diva dotyczącego pogody) stwierdzam, że podoba mi się pomysł osobnych klas do każdego z dni prognozy :)

Reply:
Potrzebowałem tego, żeby ułożyć je w grid ale faktycznie to może być przydatne :D